wiring/inc/spark_wiring_ble.h

```diff
@@ -219,6 +220,9 @@ class BleAddress {
     bool operator!=(const char* address) const;
     bool operator!=(const String& address) const;
 
+    bool valid() const;
```

Comment:
Let's be consistent in our wiring APIs. This should be `isValid()`. Optionally operator bool may also be added.

Reply:
Updated and added.